staff/bernat-casasus/portfolio/index.html

```diff
@@ -0,0 +1,65 @@
+            <title>Document</title>
+    </head>
+    <body>
+            <header>
+                <h1>Jonsson Portfolio</h1>
+            </header>
+            <main>
+                <section>
+                    <h2>Green Alternative</h2>
+                    <figure>IMG</figure>
+                    <figurecaption>
+                        <p><em>Culpa qui officia deserunt mollit anim</em></p>
+                    </figurecaption>
+                    <article>
+                        <p>The project will be released on <time datetime="2018-07-07">July 7</time>  culpa qui officia deserunt mollit a culpa qui officia deserunt mollit.</p>
+                        <p>Sunt in culpa qui officia deserunt mollit anim id est Sunt in culpa qui officia deserunt mollit anim id est Sunt in culpa qui officia deserunt mollit anim id est Sunt in culpa qui officia deserunt mollit anim id est .</p>
+                        <h3>As Hinkaun said ...</h3>
+                        <blockquote>
+                                <p>It was a bright cold day in April, and the clocks were striking thirteen.</p>
+                                <footer>
+                                    First sentence in <cite><a href="http://www.george-orwell.org/1984/0.html"><i>Nineteen Eighty-Four</i></a> by George Orwell (Part 1, Chapter 1)</cite>.
+                                </footer>
+                            </blockquote>
+                    </article>
+                </section>
+                <hr>
+                <section>
+                        <h2>Hause Building</h2>
+                        <figure>IMG</figure>
+                        <figurecaption>
+                                <p><em>Culpa qui officia deserunt mollit anim</em></p>
+                        </figurecaption>
+                        <article>
+                            <p>The project will be released on <time datetime="2018-07-07">July 7</time>  culpa qui officia deserunt mollit a culpa qui officia deserunt mollit.</p>
+                            <p>Sunt in culpa qui officia deserunt mollit anim id est Sunt in culpa qui officia deserunt mollit anim id est Sunt in culpa qui officia deserunt mollit anim id est Sunt in culpa qui officia deserunt mollit anim id est .</p>
+                            <h3>As Unai said ...</h3>
+                            <blockquote>
+                                    <p>Sunt in culpa qui officia deserunt mollit anim id est Sunt in culpa qui off.</p>
+                                    <footer>
+                                        Third sentence in <cite><a href="http://www.george-orwell.org/1984/0.html"><i>Sunt Eighty-Four</i></a> by Jhon Mike (Part 7, Chapter 2)</cite>.
+                                    </footer>
+                                </blockquote>
+                        </article>
+                    </section>
+                    <hr>
+                <footer>
+                        <address>
```

Comment:
A ul inside the address tag should be useful later on when adding a css file.

Reply:
Updated!

Reply:
Good job!